resources/sip11/ra/src/main/java/org/mobicents/slee/resource/sip11/wrappers/DialogWrapper.java

```diff
@@ -120,7 +122,8 @@
 	 * 
 	 */
 	public DialogWrapper(SipActivityHandle sipActivityHandle, SipResourceAdaptor ra) {
-		super(sipActivityHandle,ra);		
+		super(sipActivityHandle,ra);
+		this.terminateOnByeCached = true;
```

Comment:
defaulting to true value may deviate from underlying dialog value, since different constructors set different value for this flag. Maybe we could use "null" as default, and set the cached value if not "null"